lib/puppet/type/java_ks.rb

```diff
@@ -178,35 +176,35 @@ def value=(*values)
   def self.title_patterns
     [
       [
-        /^([^:]+)$/,
+        %r{^([^:]+)$},
         [
-          [ :name ]
-        ]
+          [:name],
+        ],
       ],
       [
-        /^(.*):([a-z]:(\/|\\).*)$/i,
+        %r{^(.*):([a-z]:(/|\\).*)$}i,
```

Comment:
Does this have one too many backslashes? Forgot to take away the closing one?

Reply:
nope, it is weird. because \\ are weird. checked with @DavidS